src/app/site/projects/project/project.component.html

```diff
@@ -1,33 +1,14 @@
-<div>
-  <div class="show-more flex-container" fxLayout="row" fxLayoutAlign="center center">
-    <div class="flex-item" fxFlex="25">
-      <img src="{{project.mainImage.url}}" alt="{{project.title}}" class="project-image" />
+<div class="pic" fxLayout="column" fxLayoutAlign="space-around center">
+  <div fxFlex="35"></div>
+  <div  class="container" fxFlex="40">
+      <mat-card class="mat">
+        <img src="{{project.mainImage.url}}" alt="{{project.title}}" class="image"  />
+        <div class="middle">
+        <p class="">{{project.title}}</p>
+        <p class="">{{project.shortDescription}}</p>
```

Comment:
please remove class="" if you don't use any css class, in this line and everywhere it's occurred